src/test/java/org/candlepin/controller/PoolManagerTest.java

```diff
@@ -1559,8 +1591,8 @@ public List<Pool> answer(InvocationOnMock iom) {
     @Test
     public void createPoolsForExistingSubscriptionsNoneExist() {
         Owner owner = this.getOwner();
-        PoolRules pRules = new PoolRules(manager, mockConfig, entitlementCurator,
-            mockOwnerProductCurator, mockProductCurator);
+        PoolRules pRules = new PoolRules(manager, mockConfig, entitlementCurator
+        );
```

Comment:
These dangling parens can be brought back up

Reply:
Done.